### PR TITLE
[zh-cn] sync stateful-application/basic-stateful-set.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md

### CODE REVIEW

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -42,26 +42,46 @@ following Kubernetes concepts:
...
 will have to manually provision two 1 GiB volumes prior to starting this
-tutorial.
+tutorial and
+set up your cluster so that those PersistentVolumes map to the
+PersistentVolumeClaim templates that the StatefulSet defines.
 -->
 本教程假设你的集群被配置为动态制备 PersistentVolume 卷。
-如果没有这样配置，在开始本教程之前，你需要手动准备 2 个 1 GiB 的存储卷。
+你还需要一个[默认 StorageClass](/zh-cn/docs/concepts/storage/storage-classes/#default-storageclass)。
+如果没有这样配置，在开始本教程之前，你需要手动准备 2 个 1 GiB 卷并设置集群，
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:83]:
```suggestion
如果你的集群未被配置为动态制备存储，在开始本教程之前，你需要手动制备 2 个 1 GiB 卷并设置集群，
```
中间新增了一句，那就需要翻完整了

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 2]
```diff
@@ -42,26 +42,46 @@ following Kubernetes concepts:
...
+<!--
+You should configure `kubectl` to use a context that uses the `default`
+namespace.
+If you are using an existing cluster, make sure that it's OK to use that
+cluster's default namespace to practice. Ideally, practice in a cluster
+that doesn't run any real workloads.
+
+It's also useful to read the concept page about [StatefulSets](/docs/concepts/workloads/controllers/statefulset/).
+-->
+你应该配置 `kubectl` 使用一个上下文，此上下文使用 `default` 命名空间。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:64]:
```suggestion
你应该配置 `kubectl` 的上下文使用 `default` 命名空间。
```

**Pull request merged**